### PR TITLE
Envvar to disable slash filter

### PR DESCRIPTION
**Pull request by arajasek**
Maybe we merge this? Maybe we don't.

### CODE REVIEW

File: miner/miner.go [hunk 1]
```diff
@@ -325,7 +326,9 @@ minerLoop:
 
 			if err := m.sf.MinedBlock(b.Header, base.TipSet.Height()+base.NullRounds); err != nil {
 				log.Errorf("<!!> SLASH FILTER ERROR: %s", err)
-				continue
+				if os.Getenv("LOTUS_MINER_NO_FILTER") != "_yes_" {
```
Comment (magik6k) [inline miner/miner.go:329]:
```suggestion
				if os.Getenv("LOTUS_MINER_NO_FILTER") != "_yes_i_know_i_can_and_probably_will_lose_all_my_fil_and_power_" {
```

**Review by magik6k — COMMENTED** [review]
I'd make the env var a bit more explicit in what can happen when this is set

File: miner/miner.go (hunk 1 above)
Comment (arajasek) [reply miner/miner.go:329]: Seems reasonable

File: miner/miner.go [hunk 2]
```diff
@@ -325,7 +326,9 @@ minerLoop:
 
 			if err := m.sf.MinedBlock(b.Header, base.TipSet.Height()+base.NullRounds); err != nil {
 				log.Errorf("<!!> SLASH FILTER ERROR: %s", err)
-				continue
+				if os.Getenv("LOTUS_MINER_NO_FILTER") != "_yes_i_know_i_can_and_probably_will_lose_all_my_fil_and_power_" {
```
Comment (ribasushi) [inline miner/miner.go:329]: Perhaps?

```suggestion
				if os.Getenv("LOTUS_MINER_NO_SLASHFILTER") != "_yes_i_know_i_can_and_probably_will_lose_all_my_fil_and_power_" {
```

W have a number of other `FILTER`s

**Review by ribasushi — APPROVED** [review]
LGTM

Suggested varname change

Can be merged as-is though

**Pull request merged**